docker-compose.yml

```diff
@@ -183,7 +192,7 @@ services:
       <<: *k8s-labels
     image: ccdcrepository.azurecr.io/csd-theory:1.0.160
     environment:
-      - CCDC_LICENSING_CONFIGURATION=${CCDC_LICENSING_CONFIGURATION}
+      - CCDC_LICENSING_CONFIGURATION=${CCDC_LICENSING_CONFIGURATION};
```

Comment:
So do we need the semi-colon or not?

Reply:
Thank you, well spotted! Looking through the Docker Compose documentation there is no reference or example of the semi-colon so have removed it